pam/pam_cgfs.c

```diff
@@ -2291,7 +2290,7 @@ static bool cgv2_create(const char *cgroup, uid_t uid, gid_t gid, bool *existed)
 			*existed = false;
 		else
 			*existed = true;
-		return our_cg;
+		goto chown_cgroup_procs_file;
```

Comment:
our_cg could have been false, but you're jumping to a block which unconditionally returns true.